cura/GlobalStacksModel.py

```diff
@@ -18,13 +17,18 @@ class GlobalStacksModel(ListModel):
     HasRemoteConnectionRole = Qt.UserRole + 3
     ConnectionTypeRole = Qt.UserRole + 4
     MetaDataRole = Qt.UserRole + 5
+    SectionNameRole = Qt.UserRole + 6  # For separating local and remote printers in the machine management page
```

Comment:
Perhaps rename this to `DiscoverySourceRole`?

Reply:
Renamed in e623c807c